experiments/long_runs/run2_spatially_varying_only.jl

```diff
@@ -0,0 +1,187 @@
+# RUN 2 ONLY: Spatially-Varying Canopy Height
```

Comment:
If this is going to be merged into main it would be good to rename to something like `snowy_land_varying_height.jl`